include/votca/xtp/sigma_base.h

```diff
@@ -44,7 +44,7 @@ class Sigma_base {
     Index rpamax = 0;
     double eta = 1e-3;
     std::string quadrature_scheme;  // Kind of Gaussian-quadrature scheme to use
-    Index order = 100;  // only needed for complex integration sigma default:12
+    Index order = 12;  // only needed for complex integration sigma default:12
```

Comment:
actually none of the options should have a default, because defaults should be gotten from the xml files.

Reply:
removed